src/config/requests.js

```diff
@@ -3,5 +3,6 @@ export const requests = {
     experiment: 'https://www.refine.bio/experiments/',
     search: 'https://www.refine.bio/search?search='
   },
-  email_logo: 'https://www.refine.bio/refinebio_email_logo.png'
+  email_logo: 'https://www.refine.bio/refinebio_email_logo.png',
```

Comment:
Lets make these relatives paths for anything on the site.

Reply:
Since `email_logo` is used by Slack API (see https://github.com/AlexsLemonade/refinebio-web/pull/298#discussion_r1691618768), this was updated in PR #298, which is the base PR branch for all web hook related PRs, including this one, thank you David!